docs/4.1/quickstart__archive.md

```diff
@@ -0,0 +1,279 @@
+
+## Adding Nodes to Cluster
+
+Let's add another node to the cluster. The `tctl` command below will create a single-use
+token for a node to join and will print instructions for you to follow:
+
+```bash
+$ sudo tctl nodes add
+
+The invite token: n92bb958ce97f761da978d08c35c54a5c
+Run this on the new node to join the cluster:
+teleport start --roles=node --token=n92bb958ce97f761da978d08c35c54a5c --auth-server=10.0.10.1
+```
+
+Start `teleport` daemon on a new node as shown above, but make sure to use the proper
+`--auth-server` IP to point back to your localhost.
+
+Once you do that, verify that the new node has joined the cluster:
+
+```bash
+$ tsh --proxy=localhost ls
+
+Node Name     Node ID                     Address            Labels
+---------     -------                     -------            ------
+localhost     xxxxx-xxxx-xxxx-xxxxxxx     10.0.10.1:3022
+new-node      xxxxx-xxxx-xxxx-xxxxxxx     10.0.10.2:3022
+```
+
+!!! tip "NOTE":
+    Teleport also supports static pre-defined invitation tokens which can be set in the [configuration file](admin-guide.md#adding-nodes-to-the-cluster)
+
+## Adding a node located behind NAT
+With the current setup you've only been able to add nodes that have direct access to the
+auth server and within the internal IP range of the cluster. We recommend
+setting up a [Trusted Cluster](/trustedclusters) if you have workloads split
+across different networks / clouds.
+
+Teleport Node Tunneling lets you add a node to an existing Teleport Cluster. This can be
+useful for IoT applications or for managing a couple of servers in a different network.
+
+Similar to [Adding Nodes to Cluster](/quickstart/#adding-nodes-to-cluster), use `tctl` to
+create a single-use token for a node, but this time you'll replace the auth server IP with
+the URL of the Proxy Server. In the Example below, we've replaced the auth server IP with the Proxy
+web endpoint `teleport.example.com`.
+
+```bash
+$ sudo tctl nodes add
+
+The invite token: n92bb958ce97f761da978d08c35c54a5c
+Run this on the new node to join the cluster:
+teleport start --roles=node --token=n92bb958ce97f761da978d08c35c54a5c --auth-server=teleport-proxy.example.com
+```
+
+Using the ports in the default configuration, the node needs to be able to talk to ports 3080
+and 3024 on the proxy. Port 3080 is used to initially fetch the credentials (SSH and TLS certificates)
+and for discovery (where is the reverse tunnel running, in this case 3024). Port 3024 is used to
+establish a connection to the Auth Server through the proxy.
+
+## Using Node Labels
+
+Notice the "Labels" column in the output above. It is currently not populated. Teleport lets
+you apply static or dynamic labels to your nodes. As the cluster grows and nodes assume different
+roles, labels will help to find the right node quickly.
+
+Let's see labels in action. Stop Teleport `ctrl-c` on the node we just added and restart it with the following command:
```

Comment:
'Stop Teleport on the node we just added (by pressing `ctrl+c`) and restart it`...